networkx/readwrite/json_graph/jit.py

```diff
@@ -0,0 +1,96 @@
+"""
+Read and write NetworkX graphs as JavaScript InfoVis Toolkit (JIT) format JSON.
+
+See the JIT documentation and examples at http://thejit.org
+
+Format
+------
+var json = [
+  {
+    "id": "aUniqueIdentifier",
+    "name": "usually a nodes name",
+    "data": {
+      "some key": "some value",
+      "some other key": "some other value"
+     },
+    "adjacencies": [
+    {
+      nodeTo:"aNodeId",
+      data: {} //put whatever you want here
+    },
+    'other adjacencies go here...'
+  },
+
+  'other nodes go here...'
+];
+"""
+
+#    Copyright (C) 2011-2013 by
+#    Aric Hagberg <hagberg@lanl.gov>
+#    Dan Schult <dschult@colgate.edu>
+#    Pieter Swart <swart@lanl.gov>
+#    All rights reserved.
+#    BSD license.
+
+__all__ = ['jit_graph', 'jit_data']
+
+import networkx as nx
+import json
+
+
+def jit_graph(data):
+    """Read a graph from JIT JSON.
+
+    Parameters
+    ----------
+    data : JIT JSON string
+
+    Returns
+    -------
+    G : NetworkX Graph
+    """
+    G = nx.Graph()
+    for node in data:
+        for adj in node['adjacencies']:
+            G.add_edge(node['id'], adj['nodeTo'], **adj['data'])
+            G.add_node(node['id'], **node['data'])
+    return G
```

Comment:
It looks like the node is only added if other nodes connect to it. 
Maybe a ```G.add_node(node)``` should go before the ```for adj``` loop